# Representation of the world state in the SML World

VehicleState[] vehicle_states
PedestrianState[] pedestrian_states
TrafficDemand[] traffic_demand
int32[] bus_stop_ids
int32[] bus_stop_demands
int32 time

int32[] traffic_light_nodes
bool[] traffic_light_status
VehicleSensors[] sensor_states

